packages/urls/src/urls.ts

```diff
@@ -172,27 +301,49 @@ export const CARDANO_MAINNET_DREP: Url =
 export const CARDANO_PREVIEW_DREP: Url =
     'https://trezor-cardano-preview.blockfrost.io/api/v0/dreps/';
 
-export const CHROME_URL: Url = 'https://www.google.com/chrome/';
-export const CHROME_UPDATE_URL: Url = 'https://support.google.com/chrome/answer/95414';
-export const CHROME_ANDROID_URL: Url =
-    'https://play.google.com/store/apps/details?id=com.android.chrome';
-export const EXPERIMENTAL_FEATURES_KB_URL: Url =
-    'https://trezor.io/learn/a/experimental-features-in-trezor-suite';
-export const EXPERIMENTAL_PASSWORD_MANAGER_KB_URL: Url =
-    'https://trezor.io/guides/bonus-tools/retrieve-dropbox-passwords-from-password-manager';
-export const CROWDIN_URL: Url = 'https://crowdin.com/project/trezor-suite';
+// =====================
+// 📣 PROMO
+// =====================
 
-export const TREZOR_SAFE_5_URL: Url = 'https://trezor.io/trezor-safe-5';
-export const ESHOP_KEEP_METAL_SINGLE_SHARE_URL: Url =
-    'https://trezor.io/trezor-keep-metal-single-share';
+export const TREZOR_SAFE_5_URL: Url = withPlatformUtm('https://trezor.io/trezor-safe-5');
+export const ESHOP_KEEP_METAL_SINGLE_SHARE_URL: Url = withPlatformUtm(
+    'https://trezor.io/trezor-keep-metal-single-share',
+);
 
-export const ESHOP_KEEP_METAL_MULTI_SHARE_URL: Url =
-    'https://trezor.io/trezor-keep-metal-multi-share';
+export const SUITE_REFERRAL: Url = withPlatformUtm('https://trezor.io/refer-a-friend');
 
-export const TRADING_DOWNLOAD_INVITY_APP_URL: Url = 'https://invity.onelink.me/yIY4/q7ltbnv0';
+// TODO: why the medium is suite desktop banner?
```

Comment:
Let's not touch this URL, no need for `withPlatformUtm` here. We can keep `utm_medium=suite_desktop_banner`. The new utm_medium param is mainly for the chatbot and there is no chatbot on /trezor-expert-consultation page